types/src/account_config/constants/account.rs

```diff
@@ -27,41 +24,10 @@ static SENT_EVENT_NAME: Lazy<Identifier> =
 static RECEIVED_EVENT_NAME: Lazy<Identifier> =
     Lazy::new(|| Identifier::new("ReceivedPaymentEvent").unwrap());
 
-pub fn account_balance_struct_name() -> &'static IdentStr {
-    &*ACCOUNT_BALANCE_STRUCT_NAME
-}
-
 pub fn sent_event_name() -> &'static IdentStr {
     &*SENT_EVENT_NAME
 }
 
 pub fn received_event_name() -> &'static IdentStr {
     &*RECEIVED_EVENT_NAME
 }
-
-pub fn account_struct_tag() -> StructTag {
-    StructTag {
-        address: CORE_CODE_ADDRESS,
-        module: ACCOUNT_MODULE_IDENTIFIER.clone(),
-        name: ACCOUNT_STRUCT_NAME.to_owned(),
-        type_params: vec![],
-    }
-}
-
-pub fn sent_payment_tag() -> StructTag {
```

Comment:
wait, isn't this used in json-rpc crate?

Reply:
Nope!